azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -283,6 +287,31 @@ public void setExecutorActive(final boolean isActive, final String host, final i
           "Set active action ignored. Executor is already " + (isActive ? "active" : "inactive"));
     }
     this.active = isActive;
+    if (!this.active) {
```

Comment:
It looks like active state is also tracked by Executor. Is it possible for the state to be stored in one place, instead of both Executor and FlowRunnerManager? There would be less chance of the states getting out of sync (if Executor is set but not FlowRunnerManager for example, or just before FlowRunnerManager is set) if there is just one state.

Reply:
Discussed offline, and Executor is a representation of the db row, and only used in this method, so would not be available in other code looking at isActive.